app/konnect/declarative-config.md

```diff
@@ -0,0 +1,217 @@
+configuration in JSON format instead:
+
+```sh
+deck konnect dump --konnect-email <email> --konnect-password-file <pass>.txt \
+--format json \
+--output-file examples/konnect2.yaml
+```
+
+### Make changes to configuration
+
+Make any changes you like using YAML or JSON format.
+For this example, let's add a new service.
+
+1. Generate a UUID. You can do this in any way you want; a common tool is the
+`uuidgen` utility.
+
+2. Add the following snippet to your `konnect.yaml` file, replacing the `id`
+in the example with your generated UUID:
+
+    ```yaml
+    _format_version: "0.1"
+    service_packages:
+    - name: MyService
+      versions:
+      - implementation:
+          type: kong-gateway
+          kong:
+            service:
+              host: mockbin.org
+              port: 80
+              protocol: http
+              id: 9595B5F9-3B6A-4C48-BE93-9EC1B0EA487A
+              routes:
+              - methods:
+                - GET
+                - POST
+                name: mockpath
+                paths:
+                - /mockpath
+        version: "1"
+    ```
+
+    Every service must have at least one initial version.
+
+3. Check that the format is correct and that decK recognizes that you've made
+changes:
+
+    ```sh
+    deck konnect diff --konnect-email <email> --konnect-password-file <pass>.txt
+    ```
+
+    If the format and schema is correct, decK gives you a preview of what would
+    be added to the {{site.konnect_saas}} configuration:
+
+    ```sh
+    creating service 9595B5F9-3B6A-4C48-BE93-9EC1B0EA487A
+    creating route mockpath
+    creating service-package MyService
+    creating service-version 1
+    Summary:
+      Created: 4
+      Updated: 0
+      Deleted: 0
+    ```
+
+4. If you're satisfied with the preview, sync the changes to
+{{site.konnect_saas}}:
+
+    ```sh
+    deck konnect sync --konnect-email <email> --konnect-password-file <pass>.txt
+    ```
+
+    You should see the same output again:
+
+    ```sh
+    creating service 9595B5F9-3B6A-4C48-BE93-9EC1B0EA487A
+    creating route mockpath
+    creating service-package MyService
+    creating service-version 1
+    Summary:
+      Created: 4
+      Updated: 0
+      Deleted: 0
+    ```
+
+5. Check {{site.konnect_saas}} to make sure the sync worked. You should see a
+new service named `MyService` in the
+[ServiceHub](https://konnect.konghq.com/servicehub/).
+
+    ![ServiceHub tiles](/assets/images/docs/konnect/konnect-myservice.png)
+
+
+## Add global configuration
+
+Edit global {{site.konnect_saas}} configuration with decK.
+
+Global config applies to all entities in the cluster. For example, you can
+configure proxy caching on all of your services at once with one `proxy-cache`
+plugin entry.
+
+
+1. Add a consumer to the `konnect.yaml` file:
+
+    ```yaml
+    consumers:
+     - custom_id: consumer
+       username: consumer
+       keyauth_credentials:
+       - key: apikey
+    ```
```

Comment:
Why do you need this for a global/shared config and not for ... all the other things the previous section presumably showed up how to manage?